@eugene123tw - Can you check the failed integration tests below with this PR?

FAILED tests/integration/cli/classification/test_classification.py::TestMultilabelClassificationCLI::test_otx_train[Custom_Image_Classification_EfficinetNet-B0]
FAILED tests/integration/cli/classification/test_classification.py::TestMultilabelClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_EfficinetNet-B0]
FAILED tests/integration/cli/classification/test_classification.py::TestMultilabelClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_DeiT-Tiny]
FAILED tests/integration/cli/classification/test_classification.py::TestMultilabelClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_MobileNet-V3-large-1x]
FAILED tests/integration/cli/classification/test_classification.py::TestMultilabelClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_EfficientNet-V2-S]
FAILED tests/integration/cli/classification/test_classification.py::TestHierarchicalClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_EfficinetNet-B0]
FAILED tests/integration/cli/classification/test_classification.py::TestHierarchicalClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_DeiT-Tiny]
FAILED tests/integration/cli/classification/test_classification.py::TestHierarchicalClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_MobileNet-V3-large-1x]
FAILED tests/integration/cli/classification/test_classification.py::TestHierarchicalClassificationCLI::test_otx_train_cls_decr[Custom_Image_Classification_EfficientNet-V2-S]
====== 9 failed, 90 passed, 16 skipped, 4 warnings in 1113.42s (0:18:33) =======

--- a/src/otx/algorithms/common/utils/task_adapt.py
+++ b/src/otx/algorithms/common/utils/task_adapt.py
@@ -3,6 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+import warnings
+
 import numpy as np
 
 from otx.utils.logger import get_logger
@@ -99,12 +101,17 @@ def unwrap_dataset(dataset):
     :param dataset: dataset object, an instance of a dataset.
     :return: tuple of dataset object and int, the base dataset and the number of times to repeat the dataset.
     """
-    from otx.algorithms.detection.adapters.mmdet.datasets import ImageTilingDataset
 
     times = 1
     target_dataset = dataset
-    if isinstance(target_dataset, ImageTilingDataset):
-        return target_dataset, times
+    try:
+        from otx.algorithms.detection.adapters.mmdet.datasets import ImageTilingDataset
+
+        if isinstance(target_dataset, ImageTilingDataset):
+            return target_dataset, times
+    except ImportError as err:
+        warnings.warn(f"ImageTilingDataset is not supported for {dataset}", stacklevel=2)
+        warnings.warn(f"Error: {err}, {type(err)}", stacklevel=2)
 
     while hasattr(target_dataset, "dataset"):
         if hasattr(target_dataset, "times"):

